assets/js/components/Dashboard/PlacesPage.tsx

```diff
@@ -301,20 +281,27 @@ const PlacesList: ComponentType<PlacesListProps> = ({
       )}
       <Accordion flush alwaysOpen activeKey={activeEventKeys}>
         {sortedFilteredPlaces.map((place: Place) => {
+          const isOnlyFilteredByRoute =
+            modeLineFilterValue !== MODES_AND_LINES[0] &&
+            statusFilterValue === STATUSES[0] &&
+            screenTypeFilterValue === SCREEN_TYPES[0] &&
+            (showScreenlessPlaces || !filteredPlacesHaveScreenlessPlaces);
+
           return (
-            <PlaceRow
+            <PlaceRowAccordion
               key={place.id}
               place={place}
-              eventKey={place.id}
-              onClick={handleClickAccordion}
-              classNames={isFiltered || isAlertPlacesList ? "filtered" : ""}
-              filteredLine={isOnlyFilteredByRoute ? getFilteredLine() : null}
-              defaultSort={sortDirection === 0}
-              showAnimation={
+              canShowAnimation={
                 showAnimationForNewPlaces &&
                 prevPlaceIds !== undefined &&
                 !prevPlaceIds.includes(place.id)
               }
+              dispatch={dispatch}
+              activeEventKeys={activeEventKeys}
+              sortDirection={sortDirection}
+              isFiltered={isFiltered}
```

Comment:
This prop gets passed to the accordion but it doesn't seem to be used anywhere within the component

Reply:
Good catch! Got that removed.